src/hooks/sdk/useBandwidthMetrics.tsx

```diff
@@ -16,6 +16,7 @@ interface BandwidthMetrics {
 }
 
 export function useBandwidthMetrics() {
+  console.log('this hook is deprecated and will be removed in future version, please use useMediaStreamMetrics hook instead.');
```

Comment:
Nit: Capitalize "this"? "this" -> "This".

Reply:
Good catch. Will change.

Reply:
Still I dont see it updated, hence unresolving.

Reply:
Sorry missed this one. I updated in the story book. Will update.